automation/gce-deploy/leonardo.conf

```diff
@@ -118,7 +118,6 @@ sam {
 serviceAccounts {
   # To implement your own service account provisioning, subclass org.broadinstitute.dsde.workbench.leonardo.model.ServiceAccountProvider
   # and specify your concrete class here
-  providerClass = "org.broadinstitute.dsde.workbench.leonardo.auth.DefaultServiceAccountProvider"
```

Comment:
I know this is in an auto-generated file, but I just wanted to remember that I removed it, and It doesn't seem to be used anywhere

Reply:
+1, I suspect the stuff in `gce-deploy` is probably dead code and doesn't work anyway